ledger-core-bitcoin/src/CMakeLists.txt

```diff
@@ -34,66 +34,7 @@ if (TARGET_JNI)
 endif ()
 
 # Add files to compile to the project
-set(SRC_FILES
-    # bech32
-    bitcoin/bech32/BCHBech32.cpp
-    bitcoin/bech32/BitcoinLikeBech32Factory.cpp
-    bitcoin/bech32/BTCBech32.cpp
-    # block
-    bitcoin/block/BitcoinLikeBlock.cpp
-    # database
-    bitcoin/database/BitcoinLikeAccountDatabase.cpp
-    bitcoin/database/BitcoinLikeAccountDatabaseHelper.cpp
-    bitcoin/database/BitcoinLikeBlockDatabaseHelper.cpp
-    bitcoin/database/BitcoinLikeTransactionDatabaseHelper.cpp
-    bitcoin/database/BitcoinLikeUTXODatabaseHelper.cpp
-    bitcoin/database/BitcoinLikeWalletDatabase.cpp
-    bitcoin/database/BitcoinLikeOperationDatabaseHelper.cpp
-    bitcoin/database/Migrations.cpp
-    # explorers
-    bitcoin/explorers/BitcoinLikeBlockchainExplorer.cpp
-    bitcoin/explorers/LedgerApiBitcoinLikeBlockchainExplorer.cpp
-    # factories
-    bitcoin/factories/BitcoinLikeWalletFactory.cpp
-    # io
-    bitcoin/io/BitcoinLikeInput.cpp
-    bitcoin/io/BitcoinLikeInputParser.cpp
-    bitcoin/io/BitcoinLikeOutput.cpp
-    bitcoin/io/BitcoinLikeOutputParser.cpp
-    bitcoin/io/BitcoinLikeWritableInput.cpp
-    # keychains
-    bitcoin/keychains/BitcoinLikeKeychain.cpp
-    bitcoin/keychains/CommonBitcoinLikeKeychains.cpp
-    bitcoin/keychains/P2PKHBitcoinLikeKeychain.cpp
-    bitcoin/keychains/P2SHBitcoinLikeKeychain.cpp
-    bitcoin/keychains/P2WPKHBitcoinLikeKeychain.cpp
-    bitcoin/keychains/P2WSHBitcoinLikeKeychain.cpp
-    # observers
-    bitcoin/observers/BitcoinLikeBlockchainObserver.cpp
-    bitcoin/observers/LedgerApiBitcoinLikeBlockchainObserver.cpp
-    # operations
-    bitcoin/operations/BitcoinLikeOperation.cpp
-    bitcoin/operations/BitcoinLikeOperationQuery.cpp
-    # scripts
-    bitcoin/scripts/BitcoinLikeScriptOperators.cpp
-    bitcoin/scripts/BitcoinLikeScript.cpp
-    bitcoin/scripts/BitcoinLikeInternalScript.cpp
-    # synchronizers
-    bitcoin/synchronizers/BitcoinLikeBlockchainExplorerAccountSynchronizer.cpp
-    # transactions
-    bitcoin/transactions/BitcoinLikeStrategyUTXOPicker.cpp
-    bitcoin/transactions/BitcoinLikeTransaction.cpp
-    bitcoin/transactions/BitcoinLikeTransactionBuilder.cpp
-    bitcoin/transactions/BitcoinLikeTransactionParser.cpp
-    bitcoin/transactions/BitcoinLikeUTXOPicker.cpp
-    # others
-    bitcoin/BitcoinLikeAccount.cpp
-    bitcoin/BitcoinLikeAddress.cpp
-    bitcoin/BitcoinLikeCurrencies.cpp
-    bitcoin/BitcoinLikeExtendedPublicKey.cpp
-    bitcoin/BitcoinLikeWallet.cpp
-    bitcoin/BitcoinNetworks.cpp
-)
+file(GLOB_RECURSE SRC_FILES *.cpp)
```

Comment:
This change is not part of the PR. Why do you need to do that? Also, why do we have an explicit listing here — ping @pollastri-pierre who might have that information.

Reply:
I am the one who introduces the explicit listing at the first place - It was to ease the bitcoin segregation.